### ISSUE TITLE
feat(model): publish full raw target history from Metal P32 and P1 graphs

### ISSUE THREAD

**Issue by anthony-chaudhary** [anthony-chaudhary/fak#12867]
Parent: #12349
Coordinates with: #12717
Coordinates with: #12859
Coordinates with: #12238

## Parent context
#12349 requires exact old-serial versus graph-P1 versus P4 logits/state, including raw target hidden. Device path agreement alone is insufficient.

## Current state
Frozen #12717 v4 declines captureTargetHidden in P1 admission. Its completed terminal pack reads normalized hidden and KV; existing raw residual x remains on the graph. The old P1 witness asserts capture decline.

## For / Problem / Today / Better because
For native Metal MTP qualification, expose committed raw hidden from supported P1 execution. Today capture forces the original serial path, preventing a truthful three-way oracle. Reuse the existing terminal readback pack and publication helper without new kernels.

## Why this is next
This is the smallest prerequisite to observe all three actual execution paths with raw hidden enabled.

## Working spine
Established P32 resident owner -> capture-enabled P1 -> same terminal pack includes raw x -> completed receipt -> cache commit -> rememberTargetHidden -> exact retrieval and capture-off accounting comparison.

## Core through-line
One bounded publication seam on the existing qualified P1 graph, with unchanged unsupported and diagnostic exclusions except raw target capture.

## Scope
```routing
lane: model
paths: ["internal/model/metal_prefill_hybrid.go", "internal/model/metal_prefill_hybrid_test.go"]
expected_steps: 3
```
Separate managed worktree on immutable #12717 v4, #12843 lower graph and #12859 publication dependencies. Add a focused P1 capture witness and update the old capture-decline expectation. Test source review before execution.

## Scope class
Work unit: leaf
Single S1 publication prerequisite in two existing files.

## Definition of done
- [ ] Source and immutable dependency manifest independently reviewed.
- [ ] Reproduction reaches actual missing P1 capture behavior, then unchanged focused witness passes in granted device window.
- [ ] Captured result independently reviewed and integrated with explicit-path commit.

## Gold-plating boundary
No shader, universal admission, arithmetic change, tolerance relaxation, graph-P1/P4-only parity claim, or repeated normalization experiment. Existing parent strict RED remains. Capture full admitted P32 panels and subsequent P1 rows; no unsupported geometry expansion.

## Acceptance gate
Required live Metal P32 owner, capture-enabled P1 receipt with one completion/readback and zero intermediate reads, exactly H*4 additional terminal bytes, valid committed token/raw row, and bit-identical capture-on/off logits. Preserve fail-closed terminal failure and no partial publication. Exact oldserial-versus-P1-versus-P4 numerical qualification is a separate parent witness.

## Done condition
Capture enabled before P32 publishes every committed prompt row and subsequent supported P1 row; publication occurs only after successful completed execution and complete raw shape/finite validation. No test or HIL without campaign grant.

## Witness
GOMAXPROCS=2 GOFLAGS=-p=1 go test ./internal/model -run '^TestMetalQwen35P1PublishesTargetHidden$' -count=1 -timeout=60s

## Duplicate search
Open/closed P1 hidden capture query found #12717 (existing exclusion), #12859 (old serial omission), #12853 (norm isolation), and #12720 (Vulkan capture); no matching P1 publication prerequisite.

## Problem frame
Centrality: Core
Portfolio tier: 2 serving
P1: advanced - committed raw target history has execution provenance.
P2: preserved - no performance claim; account additional terminal bytes.
P3: preserved - existing buffer, helper and bounded selector.
P4: advanced - actual route required by witness.

## Work estimate
Estimate: 1 point.

## Overall completion contribution
Contribution: 1/4 points within #12349; planning estimate only.

## Completion standard
development

## Closure binding
Explicit-path verified commit cites child and (fak model); parent remains open for original exact acceptance.

## V2 revised acceptance contract (supersedes new-P1-row-only scope)
V1 source patch 303af519cc8c09dabd4e4357992fd35d508b4010f9471c581053aff0bbdaf912 and its review remain immutable, unexecuted evidence. V2 uses a separate managed worktree and the same two owned files.

Capture enabled before P32 must publish all 32 nonempty raw rows with width H and exact original token lineage, preserving resident owner promotion into P1. Include existing raw x as the last terminal result for every admitted P32/P1 capture call, with exactly P*H*4 additional bytes and unchanged encoder/buffer/wait/readback count. Validate raw shape and all finite values before KV/cache/history mutation, then commit KV/positions and publish per-row defensive copies at base+row. Failed completion, malformed raw output or declined admission publishes nothing; accepted failure remains poisoned and does not replay.

The one focused witness compares capture-on/off graph outputs exactly, requires full initial32-row history and subsequent P1 row, and covers receipt bytes plus failure atomicity. SAME final test runs against unchanged v4 production for behavioral RED and v2 production for GREEN only after fresh independent source/budget review and serialized grant. V1 approval is not v2 execution authority.

Correction: previous strict P4 stage logs proved total36 rows and the four-token suffix only. Initial equality could compare two empty histories; no full prompt capture was measured. The numerical suffix/continuation RED remains valid. This publication prerequisite does not establish oldserial-versus-P1-versus-P4 arithmetic parity.

**Comment by anthony-chaudhary** [comment]
V2 captured required-device SAME-test RED/GREEN on native Apple M3 Pro. RED session22889 exit1 failed P32 raw-readback accounting on unchanged old production; GREEN21684 exit0 passed identical test (1 run, 0 skips each; packages0.870s/0.775s). Full P32 hidden rows, P1 continuation, exact capture-on/off logits, +32768/+1024 terminal bytes, decline/failure atomicity passed. Final source manifest1bea6430714b253a995ba1e8843b537b9a257fe14d2890619f946d89eba7b023 restored and verified; authored patch4b3562ed0b063a23a5d1768804e6b414e49a3d8883ed228dea9e809c97dc4cf5 unchanged. Logs/receipt in managed99bb1b64b62e/_scratch/issue-12867-hil/. CPU/HIL slot and lease released. Await independent evidence review/integration; no commit or closure yet. Original #12349 exact oldserial/P1/P4 numerical parity remains RED; capture qualification is not arithmetic or performance qualification.
